docs/best_practices/ecephys.rst

```diff
@@ -54,22 +54,21 @@ Relative Coordinates
 ~~~~~~~~~~~~~~~~~~~~
 
 For relative position of an electrode on a probe, use ``rel_x``, ``rel_y``, and ``rel_z`` columns of the
-:nwb-schema:ref:`sec-ElectrodeTable`. These positions will be used by spike sorting software to determine electrodes
+``ElectrodeTable``. These positions will be used by spike sorting software to determine electrodes
 that are close enough to share a neuron.
 
 
 
 Units Table
 -----------
 
-
 .. _best_practice_negative_spike_times:
 
 Negative Spike Times
 ~~~~~~~~~~~~~~~~~~~~
 
 All spike times should be greater than zero. Being less than zero implies the spikes are either trial-aligned (and
-should therefore be aligned to the ``timestamps_reference_time`` of the :nwb-schema:ref:`sec-NWBFile`) or the
+should therefore be aligned to the ``timestamps_reference_time`` of the :ref:`nwb-schema:sec-NWBFile`) or the
```

Comment:
Newest version of sphinx uses slightly different syntax for intersphinx references